CMakeLists.txt

```diff
@@ -51,6 +57,7 @@ if (${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin
   endif ()
 endif ()
 
+set_property(SOURCE src/Regulator.h PROPERTY SKIP_AUTOGEN ON)
```

Comment:
Is this some weird git shenanigans from me merging dev into the branch? I know @cchafe just worked on faust code.

Reply:
IIRC the fix by @cchafe is already in dev. But I don't see this line on dev.

Reply:
It was a workaround for when it was being moced for my builds. Might not be needed anymore, but doesn't really affect things anyway, since I don't think anyone other than me is using CMake. (It's certainly not supported in any official capacity.)